lib/markets.rb

```diff
@@ -0,0 +1,3 @@
+class Markets
```

Comment:
These data classes need to be defined as belonging to the `FarMar` module, like this:

```ruby
class FarMar::Market

end
```